miqa/core/tests/pyppeteer/test_save_decisions.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+
+from miqa.core.tasks import import_data
+
+
+@pytest.mark.pyppeteer
+async def test_save_decisions_tier_1(
+    page,
+    log_in,
+    webpack_server,
+    user,
+    project_factory,
+):
+    project = project_factory(
+        creator=user,
+        name='Demo Project',
+        import_path='samples/demo_project.csv',
+    )
+    project.update_group('tier_1_reviewer', [user])
+    import_data(project_id=project.id)
+    await log_in(user)
+
+    # Open the first scan
+    await (
+        await page.waitForXPath(f'//div[contains(@class, "col")][contains(.,"{project.name}")]')
+    ).click()
+    await page.waitFor(1_000)
+    assert await page.waitForXPath('//div[contains(.,"IXI002")]')
+    scans = await page.xpath('//ul[contains(@class, "scans")]/li')
+    await scans[0].click()
+    await page.waitFor(1_000)
+
+    # Mark the first scan as Usable
+    await (
+        await page.waitForXPath('//span[contains(@class, "v-btn__content")][contains(.,"Usable")]')
+    ).click()
+    await page.waitFor(1_000)
+
+    # Mark the second scan as questionable,
+    # get a warning that there must be a comment or chip selection,
+    # Then make a comment and try again
+    await (
+        await page.waitForXPath(
+            '//span[contains(@class, "v-btn__content")][contains(.,"Questionable")]'
+        )
+    ).click()
+    assert await page.waitForXPath(
+        '//div[contains(@class,"red--text")]'
+        '[contains(.," must have a comment or artifact selection")]'
+    )
+    await (await page.waitForXPath('//textarea[contains(@name, "input-comment")]')).type(
+        'This is my comment for this questionable scan.'
+    )
+    await (
+        await page.waitForXPath(
+            '//span[contains(@class, "v-btn__content")][contains(.,"Questionable")]'
+        )
+    ).click()
+    await page.waitFor(1_000)
+
+    # Mark all artifacts as present on the third scan and mark it as questionable
+    artifact_chips = await page.xpath('//span[contains(@class,"v-chip__content")]')
+    [(await chip.click()) for chip in artifact_chips]
```

Comment:
This is a minor nit, but this pattern is discouraged in python.
```suggestion
    for chip in artifact_chips:
        await chip.click()
```

Basically, you should only use list comprehension syntax if you are interested in its return value, not for side effects.